frontend/src/components/Accordion.vue

```diff
@@ -7,7 +7,7 @@
                 <ChevronLeftIcon v-if="!disabled" class="ff-icon" />
             </div>
         </button>
-        <div ref="content" class="ff-accordion--content" :style="{'max-height': contentHeight}">
+        <div ref="content" class="ff-accordion--content">
```

Comment:
this was giving warning in the console as we no longer use contentheight, so removed it here, as was flagging in the Audit Logs